bundles/config/org.eclipse.smarthome.config.core/src/main/java/org/eclipse/smarthome/config/core/validation/internal/ConfigDescriptionParameterValidator.java

```diff
@@ -12,20 +12,30 @@
 
 /**
  * The {@link ConfigDescriptionParameterValidator} can be implemented to provide a specific validation of a
- * {@link ConfigDescriptionParameter} and its actual value.
+ * {@link ConfigDescriptionParameter} and its value to be set.
  *
  * @author Thomas Höfer - Initial contribution
  */
 public interface ConfigDescriptionParameterValidator {
 
     /**
+     * Returns true, if the validator is to be executed for the given parameter and its given value, otherwise false.
+     *
+     * @param parameter the corresponding config description parameter
+     * @param value the value to be set for the config description parameter
+     *
+     * @return true, if the validator is to be executed for the given parameter and its given value, otherwise false
+     */
+    boolean isExecutable(ConfigDescriptionParameter parameter, Object value);
```

Comment:
Hm, but thinking a bit longer about it, do we really need this method? Why isn't `validate` enough, which takes the same parameters? From a call to `validate`, I would expect that "all is fine" is a valid answer as well. With the current implementation, `validate` is only called when you are sure that it has to complain about something...